src/middlewared/middlewared/etc_files/krb5.keytab.py

```diff
@@ -0,0 +1,41 @@
+import logging
+import os
+import base64
+
+from freenasUI.common.pipesubr import pipeopen
+
+logger = logging.getLogger(__name__)
+kdir = "/etc/kerberos"
+keytabfile = "/etc/krb5.keytab"
+ktutil_cmd = "/usr/sbin/ktutil copy"
+
+
+async def ktutil_copy(temp_keytab):
+    p = pipeopen(f'{ktutil_cmd} {temp_keytab} {keytabfile}')
```

Comment:
pipeopen is not async-friendly.

This will block the event loop. Please use middlewared.utils.run or Popen